pkg/controller/statusmanager/status_manager.go

```diff
@@ -85,6 +92,13 @@ func (status *StatusManager) set(conditions ...opv1alpha1.NetworkAddonsCondition
 		updateCondition(&config.Status, condition)
 	}
 
+	config.Status.OperatorVersion = operatorVersion
+	config.Status.TargetVersion = operatorVersion
```

Comment:
I don't really get this, they are both the same, also ObserverVersion is going to go from "" to operatorVersion so maybe you just need to show a flag saying "version reached" or the like instead of this tuple of three values that are the same.

Reply:
I got you. Operator and Target versions are always the same, but it is only implementation detail. In KubeVirt for instance you can have operator in version 2 handling operand in version 1. We don't have this functionality, right after someone updates operator, operand upgrade is triggered. However, this is implicit and could change. Does it make sense?

Now for Target and Observed, those will differ during upgrade. Upgrade tests will be introduced in following PRs.

Reply:
(Also, this API follows KubeVirt)

Reply:
Ahh upgrades it is, now I understand.